website/docs/sidebars.js

```diff
@@ -32,6 +32,18 @@ module.exports = {
         'reference/notification/client',
       ],
     },
+    {
+      type: 'category',
+      label: 'アプリ間データ連携',
+      items: [
+        'reference/deep-link/overview',
```

Comment:
`deep-link`だけ単数形なのが気になってしまう、、、

```suggestion
        'reference/deep-links/overview',
```

Reply:
https://github.com/ws-4020/mobile-app-crib-notes/pull/349/commits/3a24b864b48bed2866a1f2193f0b829a25a95992